pkg/apis/hco/v1alpha1/hyperconverged_types.go

```diff
@@ -13,7 +13,9 @@ type HyperConvergedSpec struct {
 	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
 	// Important: Run "operator-sdk generate k8s" to regenerate code after modifying this file
 	// Add custom validation using kubebuilder tags: https://book.kubebuilder.io/beyond_basics/generating_crd.html
+// +kubebuilder:validation:Enum=Always,Never,IfNotPresent
     KubeVirtImagePullPolicy	string	`json:"KubeVirtImagePullPolicy,omitempty"`
+// +kubebuilder:validation:Enum=Always,Never,IfNotPresent
```

Comment:
shouldn't these comments be lined up?

Reply:
Maybe you should run `go fmt` on this stuff?

Reply:
the same - will run the go fmt and let it format the code.

generally I did this intentionally to differentiate between the placeholder comments, which provide instructions to the user and bring no functional value and the comments that have the functional value.